packages/headless/src/composables/menuButton/createMenuButton.ts

```diff
@@ -1,125 +1,114 @@
-import { computed, ref } from "vue";
+import { computed, unref, type MaybeRef } from "vue";
 import { createBuilder } from "../../utils/builder";
 import { createId } from "../../utils/id";
-import { debounce } from "../../utils/timer";
+
+type CreateMenuButtonOptions = {
+  isExpanded: MaybeRef<boolean>;
```

Comment:
```suggestion
  isExpanded: Ref<boolean>;
```

It does not really make sense that its always opened or closed, right?